README.md

```diff
@@ -0,0 +1,386 @@
+    - Create the necessary migration files.
+      - `rails generate migration CreateUsers name:string photo:string bio:text posts_counter:integer`
+      - `rails generate migration CreatePosts title:string text:text comments_counter:integer likes_counter:integer`
+      - `rails generate migration CreateComments text:text`
+      - `rails generate migration CreateLikes`
+    - Run the necessary migration files.
+      - `rails db:migrate`
+    - Table and column names should match the ERD diagram.
+      - note: photo for users table should be a link to an image
+    - Foreign keys should be included.
+      - Pay attention that in `posts` table the column holding the foreign key to the `users` table should be named `author_id`
+    - All columns that are foreign keys should have a corresponding index.
+  - Be sure to reference the from the ERD.
+    <p align="center">
+      <img src="./public/blog_app_erd_v1_1.png" alt="Data model" width="500px" />
+    </p>
+
+- Project 2: Processing data in models & Validations and Model specs
+  - Processing data in models
+    - User model
+      - a method that returns the 3 most recent posts for a given user.
+      - a method that updates the posts counter for a user.
+    - Post model
+      - a method which returns the 5 most recent comments for a given post.
+      - a method that updates the comments counter for a post.
+      - a method that updates the likes counter for a post.
+  - Validations and Model specs
+    - Use validations for models.
+    - Write unit tests for models.
+    - Add the following validations:
+      - For the User model:
+        - Name must not be blank.
+        - PostsCounter must be an integer greater than or equal to zero.
+      - For the Post model:
+        - Title must not be blank.
+        - Title must not exceed 250 characters.
+        - CommentsCounter must be an integer greater than or equal to zero.
+        - LikesCounter must be an integer greater than or equal to zero.
+    - Add unit specs for all of models' methods and validations.
+
+- Project 3: Controllers.
+  - Use controllers to handle requests and render empty views.
+  - Implements Rails naming conventions.
+  - Use params from browser requests in a safe way.
+  - create all controllers and views, won't be processing any data yet, will add simple placeholders for the views with plain HTML.
+  - NOT use rails generate scaffold command. The point of this project is to show how to set up controllers by ourself.
+  - the app handles the URLs used in the wireframes:
+    <p align="center">
+      <img src="./public/blog_root_users.png" alt="All users page wireframe"  width="200px"  />
+      <img src="./public/blog_user_page.png" alt="Single user page wireframe"  width="200px"  />
+      <img src="./public/blog_user_all_posts.png" alt="All posts by a given user page wireframe"  width="200px"  />
+      <img src="./public/blog_single_post.png" alt="Single post page wireframe"  width="200px"  />
```

Comment:
- [ ] Same here as pointed out earlier :point_up_2: These images are not showing. Kindly make sure you have the image in your repo or simply remove them from your README.